scenarios/command/c/command_client/main.c

```diff
@@ -145,6 +166,14 @@ int main(int argc, char* argv[])
     // char response_topic[strlen(obj->client_id) + 33];
```

Comment:
Nit: Is there a reason to keep these comments here?

Reply:
It looks like we aren't going to add these to env vars, so I'll move the target client id to a macro and uncomment these

Reply:
Done!